### ISSUE TITLE
events in diferents scenes

### ISSUE THREAD

**Issue by pbrito** [pablothedolphin/Scriptable-Framework#16]
If you have a gameObject with a event raiser in one scene and a listener in another scene it only works in the editor. After you build the project the events dont work in the app;

**Comment by pablothedolphin** [comment]
Hi pbrito, what you've described here is intended by design. An event listener can only listen to an event that was raised in the same scene. The only work around for this is if you have multiple scenes loaded at once. 

For clarity, listeners **register** themselves to events on the "OnEnable" callback and **unregister** themselves on the "OnDisable" callback (basically when the listener is in the scene and active). When an event is raised, it loops through all it's **current** listeners and triggers each of their responses for them. That's why your listener in your second scene doesn't trigger from your raised event in your first scene.

Does that make sense? It sounds like you read through some of the documentation. If what I said above helped then I'll add it to the documentation to clear up the misunderstanding for others.

**Comment by pbrito** [comment]
So, how can I get a GameManager (with dontDestroyOnLoad) to listen to events from other scenes?

**Comment by pablothedolphin** [comment]
hi pbrito. I think I understand what you're doing now. I set up a blank project where I had my first scene with just a game manager with dont destroy on load set on and a listener on the same gameobject but the scene had nothing raising the event. I then loaded into a second scene with only a script that raised my event on start and it worked both in the editor and in a windows build. 

Are you sure you are listening to and raising the same event in both scenes? Also, what platform are you building to?
